src/lib/objects/index.d.ts

```diff
@@ -9,11 +10,15 @@ export interface WakuObjectArgs {
 	updateStore: (updater: (state: unknown) => unknown) => void
 
 	send: (data: unknown) => Promise<void>
+	sendTransaction?: (to: string, token: Token, fee: Token) => Promise<string>
+	estimateTransaction?: (to: string, token: Token) => Promise<Token>
+	onViewChange?: (view: string) => void
```

Comment:
I guess this is called when the view changes and at one point this is called with the argument `overview`. As mentioned in another comment it would be good to settle for the naming later and be consistent with it.

Reply:
I don't think the naming can be standardised. We can not enforce the amount of views nor their names to the waku object devs. However, I don't really thing we care as long as it is URL safe string.